rest-api-spec/src/main/resources/rest-api-spec/test/cluster.allocation_explain/10_basic.yaml

```diff
@@ -0,0 +1,75 @@
+---
+"cluster shard allocation explanation test":
+  - do:
+      # there aren't any unassigned shards to explain
+      catch: request
+      cluster.allocation_explain: {}
+
+  - do:
+      indices.create:
+        index: test
+
+  - do:
```

Comment:
Either way these are super sensitive to the cluster layout so maybe just a single comment about that is plenty.